src/refract/SerializeVisitor.cc

```diff
@@ -52,13 +59,17 @@ namespace refract
     void SerializeVisitor::visit(const IElement& e)
     {
         result.set("element", sos::String(e.element()));
+        bool sourceMap = exportSourceMap;
```

Comment:
Why the extra line? Why not directly use the `exportSourceMap` variable in https://github.com/apiaryio/drafter/pull/240/files#diff-6db1eb238e705bef43e74f8d116d7cbdR64